src/components/Button.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components'
+import arrow from '/arrow-circle.svg'
+
+export const NormalButton = () => {
+  return <StyledButton>Get started</StyledButton>
+}
+
+export const OrangeButton = () => {
+  return <StyledButton $orange>Get started</StyledButton>
+}
+
+export const StickyButton = () => {
+  return (
+    <StyledButton $sticky $mobile $orange>
+      Let&apos;s go!
+      <img src={arrow} id="arrow" alt="arrow" />
+    </StyledButton>
+  )
+}
+
+export const StickyDesktopButton = () => {
+  return (
+    <StyledButton $sticky $orange $desktop>
+      Let&apos;s get started!
+      <img src={arrow} id="arrow" alt="arrow" />
+    </StyledButton>
+  )
+}
+
+//Styled Components
+
+const StyledButton = styled.button`
```

Comment:
We are amazed to see how you bundle the styles of buttons together and utilise so many variables/props.
// Sofie & Wen